app/controllers/verify/sessions_controller.rb

```diff
@@ -20,6 +19,18 @@ def new
     end
 
     def create
+      result = idv_form.submit(profile_params)
+      analytics.track_event(Analytics::IDV_BASIC_INFO_SUBMITTED_FORM, result.to_h)
+
+      if result.success?
+        submit_idv_job
+        redirect_to verify_session_result_path
+      else
+        process_failure
+      end
+    end
+
+    def show
```

Comment:
Yeah I think we can talk about it with the rest of the team, and since LOA3 has not launched publicly yet, would you feel OK if we merged it with the logic as-is and followed up with how to handle these other edge cases?